### ISSUE TITLE
Reset workspace on Help menu

### ISSUE THREAD

**Issue by JonFerraiolo** [maqetta/maqetta#66]
How about a "Reset workspace..." command on Help menu that brings up a dialog
## that tells user:

This command will reset the user's workspace to its original state. By clicking
OK, all files in the current workspace that have been created created or
modified will be lost. 

If you have files in your workspace that you need to preserve, you should
download those files from your workspace onto your local machine (using one of
the download icons on the Files palette), and then upload those files back into
your workspace after the reset operation.

Click OK if you are sure you want to reset your workspace:

**Comment by childsb** [comment]
Do we still need this?

The concept of a workspace needs to go away.  With Orion we may have projects nested deep within folders or remote.  There wont be a single top level directory that has all the users projects.  They may be buried within a tree (retrieved from GIT or WEBDAV).

**Comment by JonFerraiolo** [comment]
Yes, we still need some way for a user to initiate a "start over". There are many ways to skin this cat, but the simplest one for the time being would be to have a `Reset workspace...` command on a menu, which would post a dialog explaining what is about to happen (and how all of their files on all projects will be deleted), making sure the OK is not selected by default. If user clicks OK, then their workspace gets rebuilt as if they were a new user.

I wouldn't attempt this for M5 at this point. Seems way too risky. What users are doing for the time being is creating new projects.

**Comment by JonFerraiolo** [comment]
We now have a URL param `resetWorkbenchSettings=1` but leaving this open to see whether we want to expose this within our UI
